torch/fx/passes/annotate_getitem_nodes.py

```diff
@@ -35,6 +35,21 @@ def annotate_getitem_nodes(graph: torch.fx.Graph) -> None:
                 elif sequence_node.type._name == "List":
                     assert len(parameterized_types) == 1
                     node.type = parameterized_types[0]
+            # Generic Alias Type
```

Comment:
This is feature work, not lint right?

Reply:
It has to be added to support the new syntax